.github/workflows/e2e_nightly.yml

```diff
@@ -1,17 +1,19 @@
 name: E2E Nightly
 
-on:
-  schedule:
-    # Run every day at 23:00 UTC
-    - cron: '0 23 * * *'
+on: [push]
+#on:
+#  schedule:
+#    # Run every day at 23:00 UTC
+#    - cron: '0 23 * * *'
 
 jobs:
   e2e-build-hw:
     name: Build SYCL, UR, run E2E
     strategy:
       matrix:
         adapter: [
-          {name: CUDA}
+          {name: CUDA, str_name: cuda, config: "--cuda- --hip"},
+          {name: L0, str_name: level_zero, config: ""}
```

Comment:
You should be able to just use one name - runners have multiple labels (both `l0` and `level_zero` are accepted) and they are not case sensitive.

Reply:
Usage is case-sensitive.